.storybook/main.ts

```diff
@@ -17,7 +17,6 @@ const config: StorybookConfig = {
     '@storybook/addon-a11y',
     '@storybook/addon-themes',
     '@storybook/addon-designs',
-    '@storybook/addon-mdx-gfm',
```

Comment:
Removed this plugin from here and from dependencies. It's needed only for migration to Storybook 8 which we already use. Storybook showed a warning about this upon startup.